custom_components/tuya_local/devices/HCT-626_dual_water_timer.yaml

```diff
@@ -0,0 +1,216 @@
+    class: enum
+    category: diagnostic
+    dps:
+      - id: 44
+        type: string
+        name: sensor
+        optional: true
+        mapping:
+          - dps_val: sunny
+            value: sunny
+            icon: "mdi:weather-sunny"
+          - dps_val: cloudy
+            value: cloudy
+            icon: "mdi:weather-cloudy"
+          - dps_val: rainy
+            value: rainy
+            icon: "mdi:weather-rainy"
+          - dps_val: snowy
+            value: snowy
+            icon: "mdi:weather-snowy"
+          - dps_val: null
+            value: unavailable
+            icon: "mdi:weather-sunny-off"
+  - entity: select
+    name: Weather delay
+    icon: "mdi:weather-cloudy-clock"
+    category: config
+    dps:
+      - id: 45
+        type: string
+        name: option
+        mapping:
+          - dps_val: "cancel"
+            value: "Off"
+          - dps_val: "1"
+            value: "1 day"
+          - dps_val: "2"
+            value: "2 days"
+          - dps_val: "3"
+            value: "3 days"
+          - dps_val: "4"
+            value: "4 days"
+          - dps_val: "5"
+            value: "5 days"
+          - dps_val: "6"
+            value: "6 days"
+          - dps_val: "7"
+            value: "7 days"
+  - entity: sensor
+    name: DPS 47
+    category: diagnostic
+    dps:
+      - id: 47
+        type: string
+        name: sensor
+        optional: true
+  - entity: number
+    name: DPS 53
+    category: diagnostic
+    dps:
+      - id: 53
+        type: integer
+        name: value
+        optional: true
+  - entity: number
+    name: DPS 101
+    category: diagnostic
+    dps:
+      - id: 101
+        type: integer
+        name: value
+        optional: true
+  - entity: number
+    name: DPS 102
+    category: diagnostic
+    dps:
+      - id: 102
+        type: integer
+        name: value
+        optional: true
+  - entity: number
+    name: DPS 103
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 103
+        type: integer
+        name: value
+        range:
+          min: 30
+          max: 3600
+        mapping:
+          - scale: 60
+            step: 30
+        unit: sec
+  - entity: number
+    name: DPS 104
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 104
+        type: integer
+        name: value
+        range:
+          min: 30
+          max: 3600
+        mapping:
+          - scale: 60
+            step: 30
+        unit: sec
+  - entity: sensor
+    name: Valve 1 state
+    category: diagnostic
+    dps:
+      - id: 105
+        type: string
+        name: sensor
+        readonly: true
+  - entity: sensor
+    name: Valve 2 state
+    category: diagnostic
+    dps:
+      - id: 106
+        type: string
+        name: sensor
+        readonly: true
+  - entity: sensor
+    name: Spray Schedule
+    category: diagnostic
+    dps:
+      - id: 108
+        type: string
+        name: sensor
+        hidden: true
```

Comment:
This didn't work how I expected it to.

Reply:
Also these don't reliably get set - other activity causes them to get sent, so maybe I need force: true?